evhtp.c

```diff
@@ -2703,6 +2771,10 @@ htp__ssl_get_scache_ent_(evhtp_ssl_t * ssl, unsigned char * sid, int sid_len, in
     evhtp_ssl_sess_t   * sess;
 
     connection = (evhtp_connection_t * )SSL_get_app_data(ssl);
+    if (connection->htp == NULL)
```

Comment:
As above. Should evhtp_unlikely be used here?

Reply:
Same as the other `scache` stuff; fairly certain it isn't even used. But I will doublecheck when I get home.